requirements.txt

```diff
@@ -93,7 +93,7 @@ datacite==1.1.2
 
 
 # Metrics
-django-elasticsearch-metrics==5.0.0
+git+https://github.com/Johnetordoff/django-elasticsearch-metrics.git@update-django-3
```

Comment:
* Reference for the future, here is the change we made https://github.com/CenterForOpenScience/django-elasticsearch-metrics/pull/80/files for
* Added to post-release tasks where we will merge the PR and update the requirements.